SimpleXML/SimpleXML.php

```diff
@@ -376,7 +376,7 @@ function simplexml_load_string ($data, $class_name = "SimpleXMLElement", $option
  * the specified class. That class should extend the
  * SimpleXMLElement class.
  * </p>
- * @return SimpleXMLElement a SimpleXMLElement or <b>FALSE</b> on failure.
+ * @return mixed a SimpleXMLElement or <b>FALSE</b> on failure.
```

Comment:
`SimpleXMLElement|false`.

Some PhpStorm meta magic should be applied to gather return type from `$className` parameter.